### ISSUE TITLE
Cannot enter 2 digits for hour or minute in input_datetime

### ISSUE THREAD

**Issue by Torok42** [home-assistant/frontend#11861]
### Checklist

- [X] I have updated to the latest available Home Assistant version.
- [X] I have cleared the cache of my browser.
- [X] I have tried a different browser to see if it is related to my browser.

### Describe the issue you are experiencing

In an input_date time which only has the hour, you cannot enter 2 digits to change the hour or the minutes. For example, try entering 12:30

On computer, we can do it if we go very fast
On a cell phone it is impossible!

![image](https://user-images.githubusercontent.com/58152695/155822157-0ba5fa17-179d-44d5-a76a-4c32f1554346.png)

### Describe the behavior you expected

Be able to enter hours and minutes with 2 digits

### Steps to reproduce the issue

In Lovelace put an input_datetime which only has the time
Try changing the time to 12:30

### What version of Home Assistant Core has the issue?

core-2022.3.0b3

### In which browser are you experiencing the issue with?

Google Chrome Version 98.0.4758.102

### Which operating system are you using to run this browser?

Windows 11

**Comment by hmmbob** [comment]
Yes, seeing this too on b4

**Comment by GigaTech** [comment]
Also in final core-2022.3.0 release.
if you click on either and mouse scroll, it changes +- one and no further, looks like it looses focus

**Comment by xBourner** [comment]
can confirm this. wanted to make another issue here but found this.
tested this on win 11 with edge and with ios app on iphone. 

i noticed when you try to type 44 it will put in 04. when you try to put it 44 again this works.
this is a little bit annoying. i hope this will be fixed soon.

**Comment by Schluggi** [comment]
Firefox on Android 11, Win11 or Ubuntu just works fine.

So it seems to be a bug related to Chromium-based browsers.

**Comment by hmmbob** [comment]
Still happening on 2022.6

**Comment by Zocker1999NET** [comment]
> Firefox on Android 11, Win11 or Ubuntu just works fine.
> So it seems to be a bug related to Chromium-based browsers.

Confirm that for Firefox on Android and Debian. The Home-Assistant app seems to use Chromium, because it is affected as well.

**Comment by steverep** [comment]
I'll try to take a look at this very soon.

**Comment by emufan** [comment]
I think it is dependent if there was a two digit input before. If yes, it is loosing focus after entering only one. If no, it is working most of the time to enter two.

30 minutes -> trying to enter 31 results in 03
03 minutes -> trying to enter 31 is mostly working

10 hours -> trying to add 11, resultes in 1
1 hours -> trying to add 11 is mostly working

**Comment by RoepLuke** [comment]
How about using the os-native date and time pickers? At least in the App this should be possible.

**Comment by steverep** [comment]
The iOS and android apps use the same code as the web.

I found the problem and will link a PR soon.

**Comment by eyalpost** [comment]
Downloaded the latest stable image and this isn't fixed there. How can I know when this is released?

**Comment by steverep** [comment]
It's in the 2022. Nine beta released last Wednesday. if you're not a beta tester, you should get the 2022.9 update this Wednesday, September 7.
